src/vs/workbench/contrib/debug/browser/debugEditorContribution.ts

```diff
@@ -714,6 +714,54 @@ export class DebugEditorContribution implements IDebugEditorContribution {
 		};
 	}
 
+	private tokenizeViewPort(model: ITextModel): void {
+		this.editor.getVisibleRanges().forEach(visibleRange => {
+			model.tokenizeViewport(visibleRange.startLineNumber, visibleRange.endLineNumber);
+			this.tokenizeRange(model, visibleRange.startLineNumber, visibleRange.endLineNumber);
+		});
+	}
+
+	private tokenizeRange(model: ITextModel, start: number, end: number, isCompleteDocument?: boolean) {
```

Comment:
I do not like that isCompleteDocument is passed here.
You can easily infer that from the `start` and `end`. So I would remove that argument.